include/libdnf5/rpm/package_query.hpp

```diff
@@ -847,19 +847,45 @@ class PackageQuery : public PackageSet {
         PackageQuery & installed,
         libdnf5::sack::QueryCmp cmp_type = libdnf5::sack::QueryCmp::GTE);
 
+    /// Filter installed packages.
+    ///
+    /// Removes from the query all packages that are not installed.
+    /// @since 5.0
+    // @replaces libdnf/sack/query.hpp:method:installed()
     void filter_installed();
 
+    /// Filter available packages.
+    ///
+    /// Removes from the query all packages that are installed.
```

Comment:
Nice, I like the way how you dealt with definition of what "available package" actually means.